R/estimate_truncation.R

```diff
@@ -188,6 +192,15 @@ estimate_truncation <- function(obs, max_truncation, trunc_max = 10,
 
   # combine into ordered matrix
   dirty_obs <- purrr::map(obs, data.table::as.data.table)
+  dirty_obs <- purrr::map(dirty_obs,
+    create_clean_reported_cases,
+      horizon = 0,
+      filter_leading_zeros = filter_leading_zeros,
+      zero_threshold = zero_threshold
+  )
+  # create_clean_reported_cases returns a data.table with a breakpoint column
+  # that we don't need
```

Comment:
Would it make sense to add a `add_breakpoints` argument to `create_clean_reported_cases` instead that defaults to `TRUE` but is here set to `FALSE`?

Reply:
Yes, that's a neater approach. See change here ed0d2ad989f52c0ea4eb63dc4ef8dd4a14733ed4.